compiler/rustc_builtin_macros/src/asm.rs

```diff
@@ -7,10 +7,10 @@ use rustc_errors::{Applicability, DiagnosticBuilder};
 use rustc_expand::base::{self, *};
 use rustc_parse::parser::Parser;
 use rustc_parse_format as parse;
-use rustc_session::lint::{self, BuiltinLintDiagnostics};
+use rustc_session::lint::{self};
```

Comment:
I'm surprised rustfmt isn't complaining about this.
```suggestion
use rustc_session::lint;
```

Reply:
That's quite odd.  I definitely have the stage0 rustfmt and the tidy commit hook, and would have expected CI to yell as well.  Probably a rustfmt bug/oversight?

Reply:
Update: rust-analyzer has a hint and automatic fix for this, but it's one of the small and hard to notice suggestions, and not applied by rustfmt